rollup.config.js

```diff
@@ -3,17 +3,18 @@ import resolve from '@rollup/plugin-node-resolve';
 import commonjs from '@rollup/plugin-commonjs';
 import typescript from 'rollup-plugin-typescript2';
 import postcss from 'rollup-plugin-postcss';
+import json from '@rollup/plugin-json';
 
 const packageJson = require('./package.json');
 
 export default {
 	input: packageJson.exports.require,
 	output: [
 		{
-			file: packageJson.exports.default,
+			dir: packageJson.exports.default,
 			format: 'esm',
 			sourcemap: true,
 		},
 	],
-	plugins: [peerDepsExternal(), resolve(), commonjs(), typescript({ useTsconfigDeclarationDir: true }), postcss()],
+	plugins: [peerDepsExternal(), resolve(), commonjs(), typescript({ useTsconfigDeclarationDir: true }), json(), postcss()],
```

Comment:
Why is this plugin needed? I don't see any JSON files added to the source in this PR.

Reply:
This plugin was needed because of dependencies in CodeFence, I believe its the `react-syntax-highlighter` dependency. Without the plugin, there's an error saying the plugin needs to be installed to import json.

Reply:
Can you format the plugins array to be one on each line and add a comment for this plugin to explain that it's required for that package?

Reply:
got it